lib/oli/groups/community_account.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Oli.Groups.CommunityAccount do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "community_accounts" do
+    belongs_to :community, Oli.Groups.Community
+    belongs_to :author, Oli.Accounts.Author
+    belongs_to :user, Oli.Accounts.User
+    field :is_admin, :boolean, default: false
+
+    timestamps(type: :utc_datetime)
+  end
+
+  @doc false
+  def changeset(community, attrs \\ %{}) do
+    community
+    |> cast(attrs, [:community_id, :author_id, :user_id, :is_admin])
+    |> validate_required([:community_id])
+    |> foreign_key_constraint(:community_id)
```

Comment:
If you have the db enforce the restraint (best way), you can actually remove this here and let the db do it rather than the ORM (it cause performance issues later)